### PR TITLE
Added name to Contributors.md file

### PR DESCRIPTION
**Pull request by dilekbaykara**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Added name to Contributors.md file

## Acceptance Criteria

Name, link, and emoji

## Type of Changes

Added emoji, name, and GitHub link to list of contributors

### CODE REVIEW

**Review by deeheber — APPROVED** [review]

**Comment by deeheber** [comment]
Pre-emptively approving so you aren't waiting for me.

In order to merge this into the `main` branch you will need to resolve the merge conflicts first. Once resolved, feel free to hit the "merge pull request" button.

Video demo here: https://the-collab-lab.slack.com/archives/C07FS2SLVEJ/p1723656888946809?thread_ts=1723548237.006659&cid=C07FS2SLVEJ

**Review by ahsanatzapier — APPROVED** [review]
Thanks for tackling the merge conflict! 🚀

**Review by jendevelops — APPROVED** [review]
🔥

**Pull request merged**
